hackathon_site/hardware/views.py

```diff
@@ -52,13 +51,22 @@ def get(self, request, *args, **kwargs):
 
 
 class OrderListView(generics.ListAPIView):
-    queryset = Order.objects.all()
+    queryset = (
+        Order.objects.all().select_related("team")
+        # Causing problems with filtering, will figure out later:
```

Comment:
Minor:

```suggestion
        # TODO: Causing problems with queryset aggregations, will figure out later:
```

The `TODO` wll make it easier to find down the line (pycharm at least has a Todo section that I look through occasionally that tracks these comments)